package.json

```diff
@@ -7,11 +7,16 @@
     "ios": "react-native run-ios",
     "lint": "eslint .",
     "start": "react-native start",
-    "test": "jest"
+    "test": "jest",
+    "build-stage-ios": "ENVFILE=.env.staging npx react-native run-ios -- --reset-cache",
+    "build-stage-android": "ENVFILE=.env.staging npx react-native run-android -- --reset-cache",
```

Comment:
After run-android, "--" is that a mistake? as it has no option with it

Reply:
Actually for having options they suggested to keep a `--` and use options like --reset-cache